domain/src/main/java/com/github/leuludyha/domain/model/library/Work.kt

```diff
@@ -28,4 +30,19 @@ data class Work(
     }
 
     override fun hashCode(): Int = id.hashCode()
+
+    fun toLoadedWork(): LoadedWork {
+        return runBlocking {
```

Comment:
Wen using it we need to make sure it doesn't block the UI thread or no extreme delay happens, otherwise we will need to pass a coroutine scope as parameter.